ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/SandboxIndexService.scala

```diff
@@ -286,53 +213,30 @@ class SandboxIndexService(
     val converter = new OffsetConverter()
     converter.toAbsolute(begin).flatMapConcat {
       case LedgerOffset.Absolute(absBegin) =>
-        ledgerSyncEvents(Some(absBegin)).collect {
-          case LedgerSyncEvent
-                .AcceptedTransaction(
-                _,
-                transactionId,
-                Some(submitter),
-                _,
-                recordTime,
-                offset,
-                _,
-                _,
-                Some(appId),
-                Some(commandId))
-              if (appId == applicationId.unwrap && parties.contains(submitter)) =>
+        ledger.ledgerEntries(Some(absBegin.toLong)).collect {
+          case (offset, t: LedgerEntry.Transaction)
```

Comment:
Hm, not sure but before we were collecting transaction only when they had an `appId` and `commandId` (see line 300 and 301). We don't do that anymore and it even seems that `LedgerEntry.Transaction` always has an application Id, submitting party, and command Id. I think we might need to change that if we want to reuse it for the IndexService+ReadService implementation (where we might not always have those fields).

Reply:
good point, but I'd consider doing that only when approaching our 2. milestone. I would start that with a gap analysis and lay out a plan of actions upfront.

Reply:
agreed 👍